apps/src/lib/levelbuilder/lesson-editor/LevelToken.jsx

```diff
@@ -124,7 +124,7 @@ class LevelToken extends Component {
 
     progressBubbleLevel['isCurrentLevel'] = false;
     progressBubbleLevel['status'] = LevelStatus.not_tried;
-    progressBubbleLevel['levelNumber'] = this.props.scriptLevel.position;
+    progressBubbleLevel['levelNumber'] = this.props.scriptLevel.levelNumber;
```

Comment:
I think this line can just be delete since activeLevel should now just have levelNumber

Reply:
When I try that, the numbers in the bubbles disappear:

![Screen Shot 2020-10-15 at 10 59 46 PM](https://user-images.githubusercontent.com/8001765/96218712-2b113580-0f3a-11eb-9520-0c62c78ef376.png)

In case it helps, this is what I see in the activeLevel when this method is called:
![Screen Shot 2020-10-15 at 11 02 26 PM](https://user-images.githubusercontent.com/8001765/96218900-90652680-0f3a-11eb-9bc7-9843b2e5a516.png)

I think this means I should leave this line as it is, and if we find a different way to supply levelNumber we could change it later.

Reply:
Ah I see. This keeps going back to my struggle to keep position on level and script level straight. Sounds good to leave it.